app/helpers/view_components/mentor_inbox.rb

```diff
@@ -0,0 +1,13 @@
+module ViewComponents
+  class MentorInbox < ViewComponent
```

Comment:
Should this be scoped the same as the component, so `ViewComponents::Mentoring::Inbox`?

Reply:
(I'm not sure what conclusions we've come to on the scoping so far)

Reply:
I like that. I was unsure about the scoping resolution too, but I _think_ we decided we wanted to do add the additional layer.

Reply:
Done

Reply:
See https://github.com/exercism/v3-project-mangement/issues/21 for the wider discussion of this. We should decide there and then document this in the CONTRIBUTING.md